TeachingRecordSystem/src/TeachingRecordSystem.Core/Dqt/Queries/FindTeachersByLastNameAndDateOfBirthQuery.cs

```diff
@@ -0,0 +1,5 @@
+using Microsoft.Xrm.Sdk.Query;
+
+namespace TeachingRecordSystem.Core.Dqt.Queries;
+
+public record FindTeachersByLastNameAndDateOfBirthQuery(string LastName, DateOnly DateOfBirth, ColumnSet ColumnSet) : ICrmQuery<Contact[]>;
```

Comment:
I've tried to stick with the DQT entity names in new queries so perhaps `FindContactsByLastNameAndDateOfBirthQuery` would be more consistent?

Reply:
or how about GetContactsByLastNameAndDateOfBirthQuery  ?

Reply:
Done